custom/conf/app.example.ini

```
@@ -1591,6 +1591,10 @@ PATH =
 ;ENABLED = false
 ;; Setting this to true will run all enabled cron tasks when Gitea starts.
 ;RUN_AT_START = false
+; Note: ``SCHEDULE`` accept formats
+;    - Full crontab specs, e.g. "* * * * * ?"
+;    - Descriptors, e.g. "@midnight", "@every 1h30m"
+; See more: https://pkg.go.dev/github.com/gogs/cron@v0.0.0-20171120032916-9f6c956d3e14
```

```suggestion
;;
;; Note: ``SCHEDULE`` accept formats
;;    - Full crontab specs, e.g. "* * * * * ?"
;;    - Descriptors, e.g. "@midnight", "@every 1h30m"
;; See more: https://pkg.go.dev/github.com/gogs/cron@v0.0.0-20171120032916-9f6c956d3e14
```

Textual comments in app.example.ini should now be preceded by *two* semicolons.